algorithms/CPlusPlus/sliding-window.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+Description: A program to find target sub string in given string 
+
+Approach: Using sliding window technique to compare every possible substring with the target string.
+Also Implementing the variable length sliding window algorithm where user can give size of window and target string of same size
+
+Time complexity: O(n)
+*/
+
+
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+void sliding(string s,string target){
+    int window_size=5;
```

Comment:
what if the `target` is larger than `windows_size`, does the program account for that ?

Reply:
we can initialize window size with length of target this will resolve both issues